docs/admin/install.rst

```diff
@@ -0,0 +1,243 @@
+
+  .. note:: If Tor connections are blocked on your network, you may need to configure Tor to use bridges in order to get a connection. For more information, see the `Anon Connection Wizard <https://www.whonix.org/wiki/Anon_Connection_Wizard>`_ documentation.
+
+
+- Open an administration terminal by selecting **Q > Terminal Emulator**, and enter the following command:
+
+  .. code-block:: sh
+
+   sudo qubes-dom0-update --enablerepo=qubes-dom0-current-testing
+
+  This will update ``dom0``, the administration VM, with some required fixes to properly update other VMs. When prompted, type **Y** to apply the updates.
+
+
+- Once the ``dom0`` update is complete, select **Q > System Tools > Qubes Update** to update other VMs. in the ``[Dom0] Qubes Updater`` window, first check ``Enable updates for qubes without known available updates``, then check all entries in the list above. Then, click **Next**. The system's VMs will be updated sequentially - this may take some time. When the updates are complete, click **Finish**.
+
+
+Copy the submission key
+-----------------------
+In order to decrypt submissions, your SecureDrop Workstation will need a copy of the secret key from your SecureDrop instance's SVS. To protect this key and preserve the air gap, you will need to connect the SVS USB to a Qubes VM with no network access, and copy it from there to ``dom0``:
+
+- First, use the network manager widget in the upper right panel to disable your network connection. These instructions refer to the ``vault`` VM, which has no network access by default, but if the SVS USB is attached to another VM by mistake, this will offer some protection against exfiltration.
+
+- Next, choose **Q > Domain: vault > vault: Files** to open the file manager in the ``vault`` VM.
+
+- Connect the SVS USB to a USB port on the Qubes computer, then use the devices widget in the upper right panel to attach it to the ``vault`` VM. There will be 3 listings for the USB in the widget: one for the base USB, one for the Tails partition on the USB, labeled ``Tails``, and a 3rd unlabeled listing, for the persistent volume. Choose the third listing.
+
+- In the the ``vault`` file manager, select **+ Other Locations**., then click the persistent volume's listiing in the right panel. It will be named ```N GB encrypted``, where N is the size of the persistent volume. Enter the SVS persistent volume passphrase to unlock and mount it. 
+
+- Open an administration terminal via **Q > Terminal Emulator**, and run the following command:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault "gpg --homedir /run/media/user/TailsData/gnupg --export-secret-keys --armor <SVSFingerprint>" > /tmp/sd-journalist.sec
+
+  where ``<SVSFingerprint>`` is the submission key fingerprint. This will copy the submission key in ASCII format to a temporary file in dom0, ``/tmp/sd-journalist.sec``. Verify the that the file starts with ``-----BEGIN PGP PRIVATE KEY BLOCK-----`` using the command:
```

Comment:
should note formatting for fingerprint, e.g., without spaces

Reply:
:heavy_check_mark: